subsys/zigbee/osif/zb_nrf_platform.c

```diff
@@ -262,6 +274,11 @@ static void zboss_thread(void *arg1, void *arg2, void *arg3)
 	zb_err_code = zboss_start_no_autostart();
 	__ASSERT(zb_err_code == RET_OK, "Error when starting ZBOSS stack!");
 
+#ifdef CONFIG_ZIGBEE_CLI_SHELL
+	zb_cli_set_stack_as_started();
```

Comment:
It seems to me that rather than setting zigbee state on the CLI we should add function to check the stack state to `zb_nrf_platform`. What do you think?